app/dashboard/migrations/0055_merge_20180417_1455.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.0.3 on 2018-04-17 21:55
```

Comment:
@kziemianek Mind deleting this merge migration, `0052_auto_20180401_1112.py`, and `0050_tool.py` ? Then move your initial data creation migration out, recreate automatic migrations via `make migrations`, and re-add your data creation migration?